tests/e2e/modelcontroller_test.go

```diff
@@ -42,26 +42,30 @@ type ModelControllerTestCtx struct {
 
 func (c *ModelControllerTestCtx) validateComponentEnabled(t *testing.T) {
 	t.Run("ModelMeshServing enabled", func(t *testing.T) {
-		c.validateComponentDeployed(t, operatorv1.Managed, operatorv1.Removed, metav1.ConditionTrue)
+		c.validateComponentDeployed(t, operatorv1.Managed, operatorv1.Removed, operatorv1.Removed, metav1.ConditionTrue)
 	})
 	t.Run("Kserve enabled", func(t *testing.T) {
-		c.validateComponentDeployed(t, operatorv1.Removed, operatorv1.Managed, metav1.ConditionTrue)
+		c.validateComponentDeployed(t, operatorv1.Removed, operatorv1.Managed, operatorv1.Removed, metav1.ConditionTrue)
 	})
 	t.Run("Kserve and ModelMeshServing enabled", func(t *testing.T) {
-		c.validateComponentDeployed(t, operatorv1.Managed, operatorv1.Managed, metav1.ConditionTrue)
+		c.validateComponentDeployed(t, operatorv1.Managed, operatorv1.Managed, operatorv1.Removed, metav1.ConditionTrue)
+	})
+	t.Run("ModelRegistry enabled", func(t *testing.T) {
+		c.validateComponentDeployed(t, operatorv1.Removed, operatorv1.Removed, operatorv1.Managed, metav1.ConditionTrue)
```

Comment:
should not this be 
```suggestion
		c.validateComponentDeployed(t, operatorv1.Removed, operatorv1.Removed, operatorv1.Managed, metav1.ConditionFalse)
```
?

Reply:
more like

```go
c.validateComponentDeployed(t, operatorv1.Managed, operatorv1.Managed, operatorv1.Managed, metav1.ConditionTrue)
```

thanks for pointing this out